kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java

```diff
@@ -135,6 +135,9 @@
 	/** Whether to turn on mail debugging */
 	public static final String MAIL_DEBUG = "mail.debug";
 
+	public static final String MAIL_OURDOMAIN = "mail.ourdomain";
```

Comment:
Mail agents often display the From but not the Reply-To. I’m concerned that “no-reply” may make users think they can’t reply, even though they can.

> On Dec 10, 2015, at 4:29 PM, Sam Ottenhoff <notifications@github.com> wrote:
> 
> In kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java <https://github.com/sakaiproject/sakai/pull/1438#discussion_r47288688>:
> 
> > @@ -135,6 +135,9 @@
> >  	/** Whether to turn on mail debugging */
> >  	public static final String MAIL_DEBUG = "mail.debug";
> >  
> > +	public static final String MAIL_OURDOMAIN = "mail.ourdomain";
> no-reply@sakai.rutgers.edu <mailto:no-reply@sakai.rutgers.edu> would be a great FROM address and would allow you Yahoo to accept your mail and would allow you to de-couple your Sakai SPF/DKIM records from your institutional SPF/DKIM records.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/sakaiproject/sakai/pull/1438/files#r47288688>.
>